lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1772,6 +1763,23 @@ export class UserAgentApplication {
         }
         return response;
     }
+
+    /**
+     * Set Authority when saving Token from the hash
+     * @param state
+     * @param inCookie
+     * @param cacheStorage
+     * @param idTokenObj
+     * @param response
+     */
+    private populateAuthtority(state: string, inCookie: boolean, cacheStorage: Storage, idTokenObj: IdToken): string {
```

Comment:
Should this be in Authority.ts?

Reply:
nah! We are setting it in UAA for UAA reference. There are many small references to different response/cache elements in UAA we want to refactor as a part of saveTokenFromHash(). If Authority.ts feels right then, we can do it.